validation.go

```diff
@@ -15,3 +15,8 @@ type Validatable interface {
 //
 // Since: 1.4
 type StringValidator func(string) error
+
+// CountValidator is a function sinature for validating counts (usually selections).
+//
+// Since: 2.2
+type CountValidator func(int) error
```

Comment:
I wonder, and sorry for back and forth, any reason this shouldn't be `IntValidator` to match `StringValidator` above? There is nothing about this type that requires it to refer to counts...

Reply:
Good point. I hadn't thought about that at all.